scipy/optimize/_highs/cython/src/_highs_wrapper.pyx

```diff
@@ -555,7 +555,7 @@ def _highs_wrapper(
 
     References
     ----------
-    .. [1] https://www.maths.ed.ac.uk/hall/HiGHS
+    .. [1] https://highs.dev/
     .. [2] https://www.maths.ed.ac.uk/hall/HiGHS/HighsOptions.html
```

Comment:
@jajhall I wanted to replace this link with the equivalent at highs.dev but all I could find was:
https://ergo-code.github.io/HiGHS/man/options.html
which is not really a substitute. Is it OK to leave this link to https://www.maths.ed.ac.uk/hall/HiGHS/HighsOptions.html or is there a better version at https://highs.dev/?

Reply:
There's no better version: our options aren't currently documented, I'm afraid. We were a bit keen to delete things from https://highs.dev/, expecting [https://ergo-code.github.io/HiGHS](https://ergo-code.github.io/HiGHS) to be populated much more rapidly.

Reply:
It only appears in this one private file, so I think I'll leave it alone. Thanks!